modules/ting/ting.entities.inc

```diff
@@ -485,22 +448,32 @@ class TingCollection extends DingEntityCollection {
     return $this->types = array_unique($types);
   }
 
+  /**
+   * Gets the number of different types of materials in the collection.
+   *
+   * @return array
+   *   Count of the various types, keyed by type name.
+   */
   function getTypes_count() {
```

Comment:
Adding visibility here would be nice while we are at it - `public` is fine.

Scrutinizer complains anyway.

Reply:
Agreed - can't do anything about the naming though